runhouse/main.py

```diff
@@ -767,14 +796,42 @@ def start(
         None,
         help="Public IP address of the server. Required for generating self-signed certs and enabling HTTPS",
     ),
+    api_server_url: str = typer.Option(
+        default="https://api.run.house",
+        help="URL of Runhouse Den",
+    ),
     default_env_name: str = typer.Option(
         None, help="Default env to start the server on."
     ),
     conda_env: str = typer.Option(
         None, help="Name of conda env corresponding to default env if it is a CondaEnv."
     ),
+    from_python: bool = typer.Option(
+        False,
+        help="Whether HTTP server started from inside a Python call rather than CLI.",
+    ),
 ):
-    """Start the HTTP or HTTPS server on the cluster."""
+    """Start the HTTP server on the cluster."""
+
+    # If server is already up, ask the user to restart the server instead.
+    server_status_cmd = "runhouse server status"
+    if cluster_name:
+        server_status_cmd = server_status_cmd + f" {cluster_name}"
+    is_server_up = subprocess.run(
```

Comment:
was confused at first bc thought this was a boolean, but really its a subprocess object

```suggestion
    result = subprocess.run(
```